docs/SOP.md

```diff
@@ -100,6 +101,17 @@ In general, the metadata record of an ontology in the OBO Foundry metadata regis
 
 For a discussion on this SOP, see [here](https://github.com/OBOFoundry/OBOFoundry.github.io/issues/1848).
 
+<a name="REACTIVATION"></a>
+## Reviving obsolete, orphaned, or inactive ontologies
```

Comment:
can you please add appropriate spacing to this to make it look nicer? it's really hard to read as is

Reply:
Sorry -- I'm not sure what you mean! Appropriate spacing where?

Reply:
one blank line after headers, one blank line before lists, proper indentation for new lines within a list. You might be able to use the linting tool prettier for auto-formatting stuff like this

Reply:
ok, done. i'm doing this on github.com so i'm not sure how to use a linting tool.